mycoffee/__main__.py

```diff
@@ -18,6 +18,11 @@ def main():
     parser.add_argument('--water-ratio', help='coefficient for the water component in the ratio', type=float)
     parser.add_argument('--water', help='amount of water in each cup (gr)', type=float)
     parser.add_argument('--cups', help='number of cups', type=int)
+    parser.add_argument(
+        '--digits',
+        help='number of digits up to which the result is to be rounded',
```

Comment:
here again.

Reply:
fixed in https://github.com/sepandhaghighi/mycoffee/pull/25/commits/bec88b6475a6ad0503bbfbe61c1492eab60a8192